File: Python/Program/Mahasiswa.py
class Mahasiswa:

	__nama = ""
	__nim = 0
	__prodi = ""
	__fakultas = ""
	
	def __init__(self, nama = 0, nim = "", prodi = "", fakultas = ""):
		self.__nama = nama
		self.__nim = nim
		self.__prodi = prodi
		self.__fakultas = fakultas

	#getter dan setter
	def setNama(self, nama):
		self.__nama = nama

	def getNama(self):
		return self.__nama

	def setNim(self, nim):
		self.__nim = nim

	def getNim(self):
		return self.__nim

	def setProdi(self, prodi):
		self.__prodi = prodi

	def getProdi(self):
		return self.__prodi

	def setFakultas(self, fakultas):
		self.__fakultas = fakultas

	def getFakultas(self):
		return self.__fakultas

File: PHP/Program/Mahasiswa.php
<?php

class Mahasiswa{

	private $nama = '';
	private $nim = '';
	private $prodi = '';
	private $fakultas = '';

	public function __construct($nama = '', $nim = '', $prodi = '', $fakultas = ''){
	}

	public function setNama($nama)
	{
		$this->nama = $nama;
	}

	public function getNama()
	{
		return $this->nama;
	}

	public function setNim($nim)
	{
		$this->nim = $nim;
	}

	public function getNim()
	{
		return $this->nim;
	}

	public function setProdi($prodi)
	{
		$this->prodi = $prodi;
	}

	public function getProdi()
	{
		return $this->prodi;
	}

	public function setFakultas($fakultas)
	{
		$this->fakultas = $fakultas;
	}

	public function getFakultas()
	{
		return $this->fakultas;
	}

}

?>